src/components/Modal.js

```diff
@@ -85,7 +85,7 @@ function Modal({
         // Add the focus update to the end of the event queue
         // Per react-focus-lock: https://github.com/theKashey/react-focus-lock#unmounting-and-focus-management
         await delay(0);
-        previouslyFocusedNode.current = modalNode.current.contains(document.activeElement) ? previouslyFocusedNode.current : document.activeElement;
+        previouslyFocusedNode.current = modalNode.current?.contains(document.activeElement) ? previouslyFocusedNode.current : document.activeElement;
```

Comment:
can you explain this change? was it just for ts conversion?

Reply:
Yup!

Reply:
Yeah, the new TS complained that it might be undefined, so it needs the `?`

Reply:
The unit test actually elucidated that the `modalNode.current` can be undefined. So, now it handles that gracefully.